package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@hawktracer/client",
+  "version": "1.0.0",
+  "description": "A library that obtains profile data generated by HawkTracer",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "repository": "https://github.com/amzn/hawktracer.git",
+  "author": "Hojin Ghim <hojin@amazon.com>",
+  "license": "MIT",
+  "scripts": {
+    "test": "jest",
+    "prepare": "tsc",
+    "install": "cmake-js compile --target nodejs_bindings_client --CDENABLE_CLIENT=YES --CDENABLE_NODEJS_BINDINGS=YES --CDBUILD_STATIC_LIB=YES"
+  },
+  "dependencies": {
+    "@types/bindings": "^1.3.0",
```

Comment:
At the moment, the typescript files are built at the time of installation, not deployment. So this will have to be installed for the dependent packages too.